cli/utilities/core_utils.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict
```

Comment:
This method is already present in `cli/utilities/utils.py` - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/utils.py#L31